src/travel-details-map-screen/TravelDetailsMapScreenComponent.tsx

```diff
@@ -287,36 +292,39 @@ const LiveVehicle = ({
             />
           </TouchableOpacity>
 
-          <View
-            style={{
-              shadowColor: '#000000',
-              shadowOffset: {width: 0, height: 2},
-              shadowOpacity: 0.2,
-              shadowRadius: 2,
-              position: 'absolute',
-              top: -Math.sin(bearingRadians) * directionArrowOffsetFromCenter,
-              left: Math.cos(bearingRadians) * directionArrowOffsetFromCenter,
-            }}
-          >
-            <ThemeIcon
-              svg={BusLiveArrow}
-              fill={
-                isError
-                  ? theme.interactive.interactive_destructive.default.background
-                  : isStale
-                  ? theme.interactive.interactive_1.default.background
-                  : fillColor
-              }
-              width={iconSize}
-              height={iconSize}
+          {pitch < 7 && ( // allow just a tiny bit of pitch before hiding the live bus direction arrow
```

Comment:
Personally I would have found this code a lot more readable if this was extracted to a separate component.

Reply:
Good point, will fix

Reply:
@KristianSelnas See new commit for this one

Reply:
I have now separated DirectionArrow into its own file in an upcoming commit. Would you also want LiveVehicle and maybe even LiveVehicleIcon to be separated into their own files?

Reply:
I can really just answer from a personal preference (which in most cases is "one component, one file). I think @gorandalum is better suited to answer what our official policy is. (I haven't checked if our coding guidelines mention this)

Reply:
I just pushed the commit with DirectionArrow in its own file. Can make another one for LiveVehicle later if desirable.